src/client/src/apps/MainRoute/widgets/blotter/components/Blotter.tsx

```diff
@@ -73,25 +72,14 @@ const Blotter: React.FC<BlotterProps> = props => {
     [gridApi],
   )
 
-  const getCurrencyPairContext = (currencyPair: string): Context => {
-    const formattedCurrencyPair = `${currencyPair.slice(0, 3)}/${currencyPair.slice(3, 6)}`
-    return {
-      type: 'fdc3.security',
-      name: formattedCurrencyPair,
-      id: {},
-      market: 'CURRENCY',
-    }
-  }
-
   const broadcastContext = (currencyPair: string) => {
-    if (!currencyPair) {
-      return
-    }
-    if (!platform.fdc3.broadcast) {
-      console.error(`Broadcasting is not available on the platform ${platform.name}`)
+    const { currencyPairSelected } = platform.intents
+
+    if (!currencyPair || typeof currencyPairSelected !== 'function') {
       return
     }
-    platform.fdc3.broadcast(getCurrencyPairContext(currencyPair))
+
+    currencyPairSelected(currencyPair)
```

Comment:
I really like this - exposing intents at a platform level seems like a great idea.

Reply:
On second thought - what we are doing here is a "context broadcast", not raising an intent, should we distinguish in the API, or is it not necessary?

Reply:
Will follow up on the other PR.